site/src/main/paradox/SideInputs.md

```diff
@@ -0,0 +1,102 @@
+# Side Inputs
```

Comment:
Should we also add a section to discourage `map` on side input as the operation is a `view` and will be applied on every element accessing the side input ?

Reply:
Hmm that seems like a better fit for the scaladocs, or a warning when using the `map` function. 

Do we need to revisit the related PRs? It looks like the intent was to cache values to reduce any lookups into the view? 

#2287 and the issues #1967 and #2101